deploymentCharts/vulnerability-analyzer/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
I don't think keeping only the `deployment.yaml` separately makes sense, especially since there are template expressions still in here that are not resolvable without the other files of the chart (e.g. `values.yaml`). 

IMO either the entire helm chart should be stored separately, or nothing at all.

Reply:
added example values in the deployment.yml